sample-apps/java/client/devserver-client-sample/README.md

```diff
@@ -0,0 +1,39 @@
+# Client Sample Project: Development Server
+
+This project demonstrates how to configure a Temporal Client to work with the local Development Server.
+
+## Configuring Clients
+
```

Comment:
This comment applies to the next few sentences. The first one suggests that Clients have some special support for a Temporal Service launched through the `temporal server start-dev` command, but that's not the case. The Client is equally capable of connecting to a Temporal Service started some other way (such as through Docker Compose). 

In some SDKs (such as Go), the hostname also has a default (`localhost`), but it's implicitly specified in Java by the call to `newLocalServiceStubs` (as opposed to calling `WorkflowServiceStubs.newInstance` and passing in an `WorkflowServiceStubsOptions` instance that specifies the target host). So, it's not really that Clients (in general) know the hostname where the Temporal Service is running, it's that you called a specific method that represents a connection to `localhost`. 

Based on that, I'd condense these three sentences into something like this:

> The `WorkflowServiceStubs` instance represents a connection to the Temporal Service. Calling the `WorkflowServiceStubs.newLocalServiceStubs()` method returns an instance that will connect to a Temporal Service running on the local machine, using the default port (7233).

Reply:
Doing exactly that. Thank you.